### PR TITLE
Make the package controller a package

### PR DESCRIPTION
**Pull request by lewisdiamond**
*Description of changes:*
Helm chart changes for package controller as a package. This removes the CRDs from the main package meaning this will break installation with the current CLI. The CLI must be updated once this is available.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by lewisdiamond** [comment]
/hold
I want to test keeping the CRDs in the main chart temporarily.

File: charts/eks-anywhere-packages/templates/package.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{{- if not .Values.workloadOnly }}
+{{-  $namespace := printf "%s-%s" "eksa-packages" .Values.clusterName -}}
+{{- if not (lookup "packages.eks.amazonaws.com/v1alpha1" "Package" "eksa-packages-main" "eks-anywhere-packages") }}
+apiVersion: packages.eks.amazonaws.com/v1alpha1
+kind: Package
+metadata:
+  annotations:
+    "helm.sh/resource-policy": keep
+  name: eks-anywhere-packages
```
Comment (jonahjon) [inline charts/eks-anywhere-packages/templates/package.yaml:9]: So moving the namespace to the CRDs would make it so upgrading itself wouldn't fail right?

**Pull request merged**